cold-wallet/wallet/cold-wallet.ts

```diff
@@ -97,6 +98,14 @@ export class ColdWallet {
     }
   }
 
+  public async signMessage(message: string): Promise<string> {
+    if (!this.wallet) throw new Error('Wallet is not initialized');
+
+    const account = await this.wallet.get(0);
```

Comment:
We'll need to be able to sign a message with every account (not only account 0) ... I would implement this already now.